frontend/src/hocs/withMapboxToken/MapboxTokenError.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement } from "react"
+import ErrorElement from "components/shared/ErrorElement"
+import {
+  MapboxTokenFetchingError,
+  MapboxTokenNotProvidedError,
+} from "hocs/withMapboxToken/MapboxToken"
+
+interface Props {
+  error: Error | MapboxTokenFetchingError | MapboxTokenNotProvidedError
+  deltaType: string
+  width: number
+}
+
+const MapboxTokenError = ({
+  error,
+  width,
+  deltaType,
+}: Props): ReactElement => {
+  if (error instanceof MapboxTokenNotProvidedError) {
+    return (
+      <ErrorElement
+        width={width}
+        name="No Mapbox token provided"
+        message={
+          <>
+            To use `st.{deltaType}` you need to set up a Mapbox access token.
+            To get a token, create an account at{" "}
+            <a href="https://mapbox.com">https://mapbox.com</a>. It's free for
+            moderate usage levels! Once you have a token, just set it using the
+            Streamlit config option `mapbox.token`. See{" "}
+            <a href="https://docs.streamlit.io/cli.html#view-all-config-options">
+              our documentation
+            </a>{" "}
+            for more info on how to set config options.
+          </>
+        }
+      />
+    )
+  }
+
+  if (error instanceof MapboxTokenFetchingError) {
+    return (
+      <ErrorElement
+        width={width}
+        name="Error fetching Streamlit Mapbox token"
+        message={
+          <>
+            This app requires an internet connection. Please when that your
```

Comment:
Typo: when → check

Also, instead of "Please check that your connection is working" just say "Please check your connection"

And use `<p>` tags here too:

```
<p>This app...</p>
<p>If you think...</p>
```